inlang/source-code/cli/src/commands/machine/translate.ts

```diff
@@ -8,9 +8,10 @@ import prompts from "prompts"
 
 export const translate = new Command()
 	.command("translate")
+	.requiredOption("--project <path>", "Path to the project settings file.", "./project.inlang.json")
```

Comment:
what about a variable consumed by commands who need it to ensure consistency without relying on global options? 

```ts
export const projectOption = ["--project <path>", "Path to the project settings file", "./project.inlang.json"]
```

Reply:
done ✅